src/components/Table.js

```diff
@@ -391,43 +400,54 @@ export default class Table extends PureComponent {
   };
 
   isInfiniteLoadingEnabled = () => {
-    return this.getUnloadedRowCount();
+    return !!this.getUnloadedRowCount();
   };
 
-  handleScroll = ({ target }) => {
+  handleScroll = () => {
     const {
       infiniteScrollThreshold,
       infiniteScrollLoadMore,
       infiniteScrollPageSize,
       data
     } = this.props;
 
-    if (this.isInfiniteLoadingEnabled()) {
-      const scrollPercentage =
-        (target.scrollTop / (target.scrollHeight - target.clientHeight)) * 100;
+    const dataCount = data.length;
+    const nextPage = Math.floor(dataCount / infiniteScrollPageSize) + 1;
 
-      if (scrollPercentage > infiniteScrollThreshold) {
-        const nextPage = data.length / infiniteScrollPageSize + 1;
+    if (!this.requestedPages[nextPage]) {
+      const targetRow = this.rowRefs[dataCount - infiniteScrollThreshold];
 
-        if (!this.requestedPages[nextPage]) {
-          this.requestedPages[nextPage] = true;
+      const {
+        top: innerTop,
+        height: viewPortHeight
+      } = this.innerRef.getBoundingClientRect();
```

Comment:
Maybe we should give access to the `innerRef` so that user can scroll to top and do some stuff like that.

Reply:
Yes. It's already accessible as `<Table ref={ref => ref.innerRef} />` cause it's an instance prop.